<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLG_EditScript</class>
 <widget class="QDialog" name="DLG_EditScript">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit FPGA</string>
  </property>
  <widget class="QSpinBox" name="spinBox_inputs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_outputs">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_inputs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Inputs</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_outputs">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Outputs</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_script">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>661</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_setScript">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>420</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Script</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_startScript">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>In vars:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_endScript">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Outvars:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_genCircuit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Circuit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_genTuthTable">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>390</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Truth Table</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_maxGenTime">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>420</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_maxGenTime">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Gen Time (s)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
